<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFFitAlgorithmErrorEstimateModeWidget</class>
 <widget class="QWidget" name="QFFitAlgorithmErrorEstimateModeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="cmbMode">
     <item>
      <property name="text">
       <string>from algorithm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bootstrapping (BS)</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labBSRepeat">
     <property name="text">
      <string>BS-repeats:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBSRepeats">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labBSFrac">
     <property name="text">
      <string>BS-fraction:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spinBSFraction">
     <property name="suffix">
      <string> %</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>60.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labBSDist">
     <property name="text">
      <string>BS-distortion:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spinBSDistortion">
     <property name="suffix">
      <string> %</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
